messages/protos/messages.proto

```diff
@@ -67,19 +67,33 @@ message ReportPlayerMessage {
 }
 
 message EmotePromptMessage {
-    string emote = 2;
+  string emote = 2;
 }
 
 message EmoteEventMessage {
-    int32 actorUserId = 1;
-    string emote = 2;
+  int32 actorUserId = 1;
+  string emote = 2;
 }
 
 message QueryJitsiJwtMessage {
   string jitsiRoom = 1;
   string tag = 2; // FIXME: rather than reading the tag from the query, we should read it from the current map!
 }
 
+message FollowRequestMessage {
+  string leader = 1;
+}
+
+message FollowConfirmationMessage {
+  string leader = 1;
+  string follower = 2;
+}
+
+message FollowAbortMessage {
+  string leader = 1;
+  string follower = 2;
+}
+
```

Comment:
I stumbled upon that fact myself but didn't give it much thought because I thought "what are the odds?". If you think, it could be a problem, I'd be willing to change messages accordingly.

Reply:
Yes, that could be cool. I stumbled upon the problem while testing myself (I opened 2 tabs, so I had the same name and got the issue)

Reply:
Fixed.